### DISCUSSION TITLE
Can I get wallet private key when I connect phantom wallet?

### DISCUSSION THREAD

**Discussion by winer328** [phantom/docs#358] [General]
Is it possible to connect phantom wallet in my dapp?

**Comment by winer328** [comment]
what's this, bro?

**Comment by winer328** [comment]
SCAM!!!

**Comment by Craig42-pixel** [comment]


**Reply by winer328** [reply]
A big scammer!!!

**Comment by adamdelphantom** [comment]
Hi @winer328,

No this is not possible. Private keys are not to be exposed to dapps under any situation.
